### PR TITLE
OBPIH-882 Add translation support

### PR DESCRIPTION
**Pull request by DominikaHatala**

### CODE REVIEW

File: src/js/components/stock-movement-wizard/CreateStockMovement.jsx [hunk 1]
```diff
@@ -92,8 +93,8 @@ const FIELDS = {
       disabled: queryString.parse(window.location.search).direction === 'INBOUND' && !props.isSuperuser,
     }),
   },
-  stocklist: {
-    label: 'Stock list',
+  stockList: {
```
Comment (pmuchowski) [inline src/js/components/stock-movement-wizard/CreateStockMovement.jsx]: change this back to stocklist

File: src/js/components/stock-movement-wizard/SendMovementPage.jsx [hunk 2]
```diff
@@ -56,28 +57,28 @@ const SHIPMENT_FIELDS = {
       return { formatValue: fieldValue => _.get(fieldValue, 'name') };
     },
   },
-  'stocklist.name': {
-    label: 'Stock List',
+  'stockList.name': {
```
Comment (pmuchowski) [inline src/js/components/stock-movement-wizard/SendMovementPage.jsx]: change this back to stocklist

**Comment by jmiranda** [comment]
@DominikaHatala Are you still working on this? If so, remove the "in progress" tag and I'll merge it.

**Comment by DominikaHatala** [comment]
@jmiranda I'll be working on partial receiving and putaway pages so we should wait with merging it until the rest of react pages are completed. I'll leave "in progress" label.

File: src/js/utils/form-utils.jsx [hunk 3]
```diff
@@ -18,13 +19,14 @@ export const renderFormField = (fieldConfig, fieldName, props = {}) => {
   );
 };
 
-export const renderField = ({
+const renderField2 = ({
```
Comment (pmuchowski) [inline src/js/utils/form-utils.jsx]: rename this component, maybe something like "renderFormField"

File: src/js/utils/form-utils.jsx [hunk 4]
```diff
@@ -1,9 +1,10 @@
 import React from 'react';
 import { Tooltip } from 'react-tippy';
 import PropTypes from 'prop-types';
-import { Translate } from 'react-localize-redux';
+import { getTranslate, Translate } from 'react-localize-redux';
 
 import 'react-tippy/dist/tippy.css';
+import connect from 'react-redux/es/connect/connect';
```
Comment (pmuchowski) [inline src/js/utils/form-utils.jsx]: you can use this: import { connect } from 'react-redux';
also move it before the css import (like it's done on other pages)

**Pull request merged**